examples/storage_systems/update_resource.tf

```diff
@@ -3,30 +3,32 @@ provider "oneview" {
   ov_password   = var.password
   ov_endpoint   = var.endpoint
   ov_sslverify  = var.ssl_enabled
-  ov_apiversion = 2600
+  ov_apiversion = 2800
   ov_ifmatch    = "*"
 }
 
 variable "hostname" {
   type    = string
-  default = "<storage_system_ip>"
+  default = "172.18.11.12"
```

Comment:
please remove the confidential items from this file.

Reply:
Done